DrinkManagerWeb/Models/ViewModels/DrinkCreateViewModel.cs

```diff
@@ -0,0 +1,49 @@
+﻿using BLL;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.ComponentModel.DataAnnotations;
+
+namespace DrinkManagerWeb.Models.ViewModels
+{
+    public class DrinkCreateViewModel
+    {
+        public string Id { get; set; }
+
+        [Required]
+        [StringLength(25)]
+        public string Name { get; set; }
+        [Required]
+        [StringLength(25)]
+        public string Category { get; set; }
+        public string AlcoholicInfo { get; set; }
+        [StringLength(25)]
+        public string GlassType { get; set; }
```

Comment:
GlassType is not required. I think it should be.

Reply:
done